src/nodeViews/createReactNodeViewConstructor.tsx

```diff
@@ -250,5 +297,5 @@ export function createReactNodeViewConstructor(
     };
   }
 
-  return nodeViewConstructor;
+  return Object.assign(nodeViewConstructor, { [REACT_NODE_VIEW]: true });
```

Comment:
super nit: just writing `nodeViewConstructor[REACT_NODE_VIEW] = true` might be easier to understand.

Reply:
Yeah, Typescript just likes this better haha

Reply:
I figured. 😆